tsconfig.json

```diff
@@ -6,8 +6,6 @@
     "baseUrl": ".",
     "typeRoots": ["./node_modules/@types", "./types"],
     "paths": {
-      "@techboi/utils": ["packages/utils/src"],
-      "@techboi/react": ["packages/react/src"],
```

Comment:
Should we add the new `@consent-manager/` packages here?

Reply:
No glob support and so this will get out of sync one day. :/